central/productusage/store/cache/cache.go

```diff
@@ -36,10 +39,13 @@ func NewCache() Cache {
 		lastKnown:   maputil.NewSyncMap[string, *storage.SecuredUnits](),
 		nodesMap:    maputil.NewMaxMap[string, int64](),
 		cpuUnitsMap: maputil.NewMaxMap[string, int64](),
+		mux:         &sync.Mutex{},
```

Comment:
1. `mux` does not need to be pointer
2. `sync.Mutex` fields don't have to be explicitly set up